app/routes/_auth.login.tsx

```diff
@@ -167,21 +166,23 @@ export default function Component() {
               showLabel={false}
               centerText
             />
-            <button type="submit" disabled={isPending}>
+            <button className="w-full text-center font-normal focus:outline-none focus:ring-4 py-3 md:text-base bg-warm text-white focus:ring-red-300 hover:bg-red-700 rounded-full">
```

Comment:
Could we use our re-usable primary button here? Or is it different?

Reply:
Unfortunately, they're different. Considered reusable ones but it makes them a bit complex. When it comes to colors, I found the warm color brighter than the primary. What do you think? 🙂